bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -258,6 +286,40 @@ impl Client {
             Ok(Arc::new(session_verification_controller))
         })
     }
+
+    /// Log out the current user
+    pub fn logout(&self) -> anyhow::Result<()> {
+        RUNTIME.block_on(async move {
+            _ = self.client.logout().await;
+            Ok(())
```

Comment:
Why not return the result in case of an error here?

```suggestion
            self.client.logout().await
```